codex-ui/src/styles/dimensions.pcss

```diff
@@ -23,6 +23,7 @@
    * Layout
    */
   --layout-content-width: 700px;
+  --layout-block-width: 268px;
```

Comment:
left is 300. there are also 16 px paddings 
<img width="929" alt="image" src="https://github.com/codex-team/notes.web/assets/3684889/f033300a-a1b9-487e-9ada-0cc2d504ff58">